modules/codelite/tests/test_codelite_additional_rules.lua

```diff
@@ -213,15 +213,15 @@ test2.obj: test2.rule
 		buildinputs { "toto.txt", "extra_dependency" }
 		buildoutputs { "toto.c" }
 		buildcommands { "test", "test toto.c" }
-		buildmessage "Some message"
+		buildmessage '"Some message"'
```

Comment:
You should avoid hijacking existing tests, you should create a new test explicitly for testing the escaping.

Reply:
OK, I will do that.

Reply:
Roll backed.
Dedicated test done.